### ISSUE TITLE
Occasional crash when spitting a key after save/load.

### ISSUE THREAD

**Issue by DDR0** [frogatto/frogatto#671]
![image](https://github.com/frogatto/frogatto/assets/862627/d1c74041-4909-478c-b6ae-7daa01426fef)

Keys seem to get their colour nulled out sometimes.

**Comment by al0fdf** [comment]
Sometimes? I'd say always.
[save4.txt](https://github.com/frogatto/frogatto/files/11933592/save4.txt)

Here's a reproduction case from Fan House. Load the file, go through the door, then climb down to collect the red key,. Climb back up, save your game, then exit to the titlescreen and load. Crashes for me every time.

**Comment by rkettering** [comment]
Further evidence:  in said repro case, we get the following:

```	"serialized_objects": {
		"character": [
			{
				"_uuid": "a0d0d21492f04b1994787e5895b9f9f0",
				"always_active": true,
				"current_frame": "normal",
				"custom": true,
				"label": "_2946ad99",
				"property_data": {
					"_preexisting_object": true,
					"number_of_cycles_we_have_been_underwater": 0,
					"original_label": "_507b6d01",
					"original_level": "crevice-village-house-4-backroom.cfg",
					"original_level_we_spawned_on": "crevice-village-house-4-backroom.cfg",
					"original_position": {
						"x": -640,
						"y": 832
					}
				},
				"type": "padlock_key",
				"x": -662,
				"y": 810
			},
```

However, when I specifically spit the key out, grab it again, and hit the save totem a second time, I get:

```		"serialized_objects": {
		"character": [
			{
				"_uuid": "0e12ba42c00344fd8c7c1c4f95cc50b0",
				"always_active": true,
				"created": true,
				"current_frame": "normal",
				"custom": true,
				"frame_obj": {
					"accel_y": 80,
					"body_area": "all",
					"duration": -1,
					"id": "normal",
					"image": "props/keys.png",
					"rect": [48,0,63,15]
				},
				"label": "_64314fe9",
				"property_data": {
					"_preexisting_object": true,
					"color": "red",
					"number_of_cycles_we_have_been_underwater": 0,
					"original_label": "_507b6d01",
					"original_level": "crevice-village-house-4-backroom.cfg",
					"original_level_we_spawned_on": "crevice-village-house-4-backroom.cfg",
					"original_position": {
						"x": -640,
						"y": 832
					}
				},
				"type": "padlock_key",
				"x": 1154,
				"y": -167
			},

```

So it's something about the save that happens RIGHT after exiting the door, without having spat out the key, that triggers this.

**Comment by DDR0** [comment]
Fixed in [aa76e27](https://github.com/frogatto/frogatto/commit/aa76e27f2b6670ed54facf23fd73d8a9dcd4766a#diff-e390e74331643fd19f167734d8b6629a5cce0f03bfc0fa2d1234075ad5991cbeR71).
